tools/cli/internal/openapi/filter/filter.go

```diff
@@ -105,8 +105,9 @@ func FiltersToGetVersions(oas *openapi3.T, metadata *Metadata) []Filter {
 func FiltersToCleanupRefs(oas *openapi3.T) []Filter {
 	return []Filter{
 		&TagsFilter{oas: oas},
-		&SchemasFilter{oas: oas},
+		&ResponseFilter{oas: oas},
 		&ParametersFilter{oas: oas},
+		&SchemasFilter{oas: oas},
```

Comment:
The order here is important since schemas can be referenced in unused parameters or responses, so we need to filter them before